examples/offline_inference.py

```diff
@@ -11,7 +11,7 @@
 sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
 
 # Create an LLM.
-llm = LLM(model="facebook/opt-125m")
```

Comment:
Is this change necessary? Otherwise, can we revert it? This does not look nicer than the current one.